tst/EnergyPlus/unit/OutputProcessor.unit.cc

```diff
@@ -3595,8 +3595,9 @@ namespace EnergyPlus {
 			GenOutputVariablesAuditReport();
 
 			std::string errMsg = delimited_string ({
-				"   ** Warning ** The following Report Variables were requested but not generated",
-				"   **   ~~~   ** because IDF did not contain these elements or misspelled variable name -- check .rdd file",
+				"   ** Warning ** The following Report Variables were requested but not generated -- check.rdd file",
+				"   **   ~~~   ** Either the IDF did not contain these elements, the variable name is misspelled,",
+				"   **   ~~~   ** or the requested variable is an advanced output which requires Output : Diagnostics, DisplayAdvancedReportVariables;",
```

Comment:
This is OK. I have just seen where string like this were commented out in unit tests and I thought there was a problem with Mac or Linux builds.